src/daft-core/src/series/ops/agg.rs

```diff
@@ -276,4 +276,78 @@ impl Series {
             ))),
         }
     }
+
+    pub fn bool_and(&self, groups: Option<&GroupIndices>) -> DaftResult<Self> {
+        use crate::array::ops::DaftBoolAggable;
+        match self.data_type() {
+            DataType::Boolean => {
+                let downcasted = self.bool()?;
+                match groups {
+                    Some(groups) => {
+                        Ok(DaftBoolAggable::grouped_bool_and(downcasted, groups)?.into_series())
```

Comment:
Use object syntax here instead of the fully-qualified syntax. I.e., instead of:

```rs
Ok(DaftBoolAggable::grouped_bool_and(downcasted, groups)?.into_series())
```

do the following:

```rs
Ok(downcasted.grouped_bool_and(groups)?.into_series())
```